pillar/components/samples/generic_config.ini

```diff
@@ -8,6 +8,7 @@ controller.primary_mc.ip=10.0.0.1
 controller.secondary_mc.ip=10.0.0.2
 controller.user=manage
 controller.secret='!manage'
+controller.type=
```

Comment:
@dmitrydb i guess that field is not supported currently by ConfigureSetup command. It should pass the static validation and be set to o pillar then.

Reply:
I checked configure_setup command with updated by Yash **generic_config.in**i. This template successfully passes validation of **StorageEnclosureParamsValidation**. If you want I can add it to params validation logic and set it as mandatory param for **storage_enclosure** section